test/click.spec.ts

```diff
@@ -30,7 +31,7 @@ describe('Page.click', function () {
 
     await page.goto(server.PREFIX + '/input/button.html');
     await page.click('button');
-    expect(await page.evaluate(() => result)).toBe('Clicked');
+    expect(await page.evaluate(() => globalThis.result)).toBe('Clicked');
```

Comment:
this is to keep TypeScript happy - it's happy for you to do `globalThis.X` but not just `X` or `window.X`.

Reply:
Ack. I actually think this is better since it's more explicit.